exercises/rotational-cipher/rotational_cipher_test.exs

```diff
@@ -0,0 +1,80 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("rotational_cipher.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule RotationalCipherTest do
+  use ExUnit.Case
+
+  #@tag :pending
+  test "rotate a by 1" do
+    plaintext = "a"
+    shift = 1
+    assert RotationalCipher.rotate(plaintext, shift) == "b"
+  end
+
+  @tag :pending
+  test "rotate a by 26, same output as input" do
+    plaintext = "a"
+    shift = 26
+    assert RotationalCipher.rotate(plaintext, shift) == "a"
+  end
+
+  @tag :pending
+  test "rotate a by 0, same output as input" do
+    plaintext = "a"
+    shift = 26
```

Comment:
I agree. Actually, it's a typo in the canonical test data: https://github.com/exercism/x-common/blob/master/exercises/rotational-cipher/canonical-data.json#L21. I just took that and made a quick vim macro to convert each test's hash into an Elixir test.

I'll fix the shift to match the test description, since that seems clearer than understanding that 26 is the same as 0 due to modulus.